### DISCUSSION TITLE
Certain @default value leads to repeated migrations generated

### DISCUSSION THREAD

**Discussion by TomiHiltunen** [prisma/prisma#9817] [Q&A] [answered]
**Hi! 👋**

I have a schema with a custom `@default` value which sets a `DateTime` field x days into the future. This definition leads to all following migrations to contain an alter table statement for the default value. I would be delighted if someone can point out what I could do differently to avoid this issue.

**Schema:**
```prisma
model Invitation {
  ...
  expiresAt      DateTime             @default(dbgenerated("NOW() + INTERVAL '7 DAYS'"))
  ...
}
```

**Resulting line in all the following migrations:**
```sql
-- AlterTable
ALTER TABLE "Invitation" ALTER COLUMN "expiresAt" SET DEFAULT NOW() + INTERVAL '7 DAYS';
```

**Comment by ryands17** [comment, marked as answer]
@TomiHiltunen 👋 

This seems like a bug. It would be great if you could create an issue [here](https://github.com/prisma/prisma/issues/new?assignees=&labels=kind%2Fbug&template=bug_report.yml) so that we can look into this.

**Reply by TomiHiltunen** [reply]
Thanks, I will create one. :+1:

**Reply by TomiHiltunen** [reply]
Created: https://github.com/prisma/prisma/issues/9823

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
